src/DynamoCore/Graph/Nodes/NodeModelExtensions.cs

```diff
@@ -99,6 +99,9 @@ internal static List<IGraphicItem> GeneratedGraphicItems(this NodeModel node, En
 
         internal static string GetOriginalName(this NodeModel node)
         {
+            if (node == null)
+                return string.Empty;
+
```

Comment:
Refer to [coding standard](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards) for a recommendation on single-line statements.